db/migrate/20160308100307_add_filtered_api_to_ovirt.rb

```diff
@@ -0,0 +1,5 @@
+class AddFilteredApiToOvirt < ActiveRecord::Migration
+  def change
+    add_column :compute_resources, :filtered_api, :boolean, :default => false, :null => false
```

Comment:
you can use the serialized attribute instead of adding db columns that do not fit other compute resources?